source/technical/ep.rst

```diff
@@ -0,0 +1,435 @@
+	                "clock_speed": 3100000000, 
+	                "instruction_set": "x86-64", 
+	                "model": "Intel Xeon", 
+	                "other_description": "Intel(R) Xeon(R) CPU E5-2670 v3 @ 2.30GHz", 
+	                "vendor": "Intel", 
+	                "version": "E5-2670 v3"
+	            }, 
+	            "storage_devices": [
+	                {
+	                    "device": "sda", 
+	                    "driver": "megaraid_sas", 
+	                    "humanized_size": "250 GB", 
+	                    "interface": "SATA", 
+	                    "model": "ST9250610NS", 
+	                    "rev": "AA63", 
+	                    "size": 250059350016, 
+	                    "vendor": "Seagate"
+	                }
+	            ], 
+	            "supported_job_types": {
+	                "besteffort": false, 
+	                "deploy": true, 
+	                "virtual": "ivt"
+	            }, 
+	            "type": "node", 
+	            "uid": "b0525159-5c95-4b71-83f2-b8d6bdd2acd2", 
+	            "version": "53c90ef0512d5013ee30d431cd62e68bfd34d4ca"
+	        }
+	    ], 
+	    "hardware_page": 0, 
+	    "hardware_page_size": 25, 
+	    "metric_page": 0, 
+	    "metric_page_size": 25, 
+	    "metrics": [
+	        {
+	            "instance_id": "44ad06ee-41d7-48f9-a52a-179030754707", 
+	            "metric_id": "dd22e02386714516a913d966659617eb", 
+	            "metric_name": "interface-eno1@if_dropped"
+	        }, 
+	        {
+	            "instance_id": "44ad06ee-41d7-48f9-a52a-179030754707", 
+	            "metric_id": "512ac94754b64906a12960d1f0a929c9", 
+	            "metric_name": "interface-eno1@if_errors"
+	        },
+	         
+	        ...
+            
+	        {
+	            "instance_id": "44ad06ee-41d7-48f9-a52a-179030754707", 
+	            "metric_id": "89cec271c0fb477b9e7bb37ad3df1331", 
+	            "metric_name": "memory@memory.slab_recl"
+	        }
+	    ],  
+	    "report_time_in_ct": "2018-10-19 11:06:51", 
+	    "report_time_in_utc": "2018-10-19 16:06:51", 
+	    "site": "CHI_DEV_UC", 
+	    "testbed_version": "702d4d47ab21c890c0bb146f4e0256f618264487"
+	}
+	
+The ``events`` section is a list of testbed events in the order of event timestamp. The ``hardware`` section contains information of all the nodes that were used in the experiment. 
+The hardware information is retrieved by using the same method as :ref:`the Resource Discovery <resource-discovery>`. The ``metrics`` section is a list of the metrics captured during the experiment. 
+The *Experiemnt Precis* only contains the ``instance_id``, ``metric_id``, and ``metric_name`` in the ``metrics`` list. You can use :ref:`the openstack metric command line <retrieve-metric>` 
+to get all the measurements of a particular metric over time for an instance. 
+	
+For more information, run:
+
+.. code-block:: bash
+
+   cep print --help
+   
+.. important::
+   Chameleon only keeps experiment precis that are created within **180 days**. 
```

Comment:
I have no idea what the plural of "precis" is... but this should be plural.

Alternatively:
```suggestion
   Chameleon only keeps an experiment precis for **180 days**. 
```